mypy/checkmember.py

```diff
@@ -109,10 +109,19 @@ def analyze_member_access(name: str,
     elif isinstance(typ, NoneTyp):
         if chk.should_suppress_optional_error([typ]):
             return AnyType(TypeOfAny.from_error)
-        # The only attribute NoneType has are those it inherits from object
-        return analyze_member_access(name, builtin_type('builtins.object'), node, is_lvalue,
-                                     is_super, is_operator, builtin_type, not_ready_callback, msg,
-                                     original_type=original_type, chk=chk)
+        is_python_2 = chk.options.python_version[0] == 2
+        # In Python 2 "None" has exactly the same attributes as "object". Python 3 adds a single
+        # extra attribute, "__bool__".
+        if not is_python_2 and name == '__bool__':
```

Comment:
It would read more natural if you use `if is_python_3 ...` here and define it as `is_python_3 = chk.options.python_version[0] >= 3` above.

Reply:
> It would read more natural if you use `if is_python_3 ...` here and define it as `is_python_3 = chk.options.python_version[0] >= 3` above.

Done. Though now `is_python_3` will be True on Python 4. 😸 
Thanks!